.github/workflows/ci.yml

```diff
@@ -112,3 +112,15 @@ jobs:
       - run: pnpm test:all
         env:
           'EMBER_CLI_${{ matrix.feature-flag }}': true
+
+  deprecations-broken-test:
+    name: Debug and Prebuilt with Deprecations as Errors
```

Comment:
This name is copy and pasted from ember-source?

Reply:
Yes, Isn't it correct one?

Reply:
Changes the name. Thank you for pointing it out